cortex-m-rt/examples/override-exception.rs

```diff
@@ -4,20 +4,18 @@
 #![no_main]
 #![no_std]
 
-extern crate cortex_m;
-extern crate cortex_m_rt as rt;
 extern crate panic_halt;
 
-use cortex_m::asm;
-use rt::{entry, exception, ExceptionFrame};
+use cortex_m::{asm, peripheral::scb::Vector};
```

Comment:
I can't see `Vector` in https://docs.rs/cortex-m/latest/src/cortex_m/peripheral/scb.rs.html?search=Vector#19-98?

Reply:
What do you mean? I just added/renamed it in this PR.

Reply:
Oh, brain fart. My bad.